Text/Pandoc/Builder.hs

```diff
@@ -305,6 +305,16 @@ setDate = setMeta "date"
 
 -- Inline list builders
 
+-- | Extend a list with a default element up to a maximum length
```

Comment:
we aren't using `doctest` in this project... so I'd just add a few more tests to `test-pandoc-types.hs`